src/apis/fetchMovie.ts

```diff
@@ -0,0 +1,39 @@
+import TMDBResponse from "../types/TMDBResponse";
+
+const options = {
+  method: "GET",
+  headers: {
+    accept: "application/json",
+    Authorization: `Bearer ${process.env.ACCESS_TOKEN}`,
+  },
+};
+
+const isValidPage = (page: number) =>
+  Number.isInteger(page) && 1 <= page && page <= 500;
+
+export const fetchPopularMovie = async (
+  page: number
+): Promise<TMDBResponse | null> => {
+  if (!isValidPage(page)) return null;
+
+  const url = `https://api.themoviedb.org/3/movie/popular?language=ko-KR&page=${page}`;
+
+  const res = await fetch(url, options);
+  const tmdbResponse: TMDBResponse = await res.json();
+
+  return tmdbResponse;
+};
+
+export const fetchTargetMovie = async (
+  page: number,
+  movieName: string
+): Promise<TMDBResponse | null> => {
+  if (!isValidPage(page)) return null;
+
+  const url = `https://api.themoviedb.org/3/search/movie?query=${movieName}&include_adult=false&language=ko-KR&page=${page}`;
+
+  const res = await fetch(url, options);
+  const tmdbResponse: TMDBResponse = await res.json();
+
+  return tmdbResponse;
```

Comment:
```suggestion
  return await res.json() as TMDBResponse;
```

위와 같이 한 줄로 작성해도 되지 않을까요?